app/policies/claims/support_user_policy.rb

```diff
@@ -4,6 +4,6 @@ def destroy?
   end
 
   def remove?
-    destroy?
+    user.support_user?
   end
```

Comment:
This is a good catch, but I think it should go into the `destroy?` method, no?

```ruby
def destroy?
  user != record && user.support_user?
end
```

We can remove `remove?`, as it's already defined in `Claims::ApplicationPolicy`.